critiquebrainz/frontend/templates/macros.html

```diff
@@ -96,7 +96,7 @@
 {% macro show_tags(tags) %}
   {% if tags %}
     {% for tag in tags %}
-      <a href="{{ 'https://musicbrainz.org/tag/%s'|format(tag['name']) }}">{{ tag['name'] }}</a>
+      <a href="{{ 'https://musicbrainz.org/tag/%s'|format(tag) }}">{{ tag }}</a>
```

Comment:
Is there any reason to use `format` instead of just outputting the value directly into the template?